craft_parts/plugins/rust_plugin.py

```diff
@@ -53,9 +51,29 @@ class RustPluginProperties(PluginProperties, PluginModel):
     rust_channel: Optional[str] = None
     rust_use_global_lto: bool = False
     rust_no_default_features: bool = False
+    rust_ignore_toolchain_file: bool = False
+    rust_verbose_output: bool = False
```

Comment:
Certain plugins have the `<plugin name>-parameters` option to append to the relevant command line, can we use that for verbose output instead? We could use `rust-cargo-parameters` or similar and append to `config_cmd` below, and this could be used for other simple parameters.

Reply:
Done